rocky/reports/views/base.py

```diff
@@ -79,7 +79,7 @@ def get_context_data(self, **kwargs):
 class BaseReportView(OOIFilterView):
     def setup(self, request, *args, **kwargs):
         super().setup(request, *args, **kwargs)
-        self.selected_oois = list(set(request.GET.getlist("ooi", [])))
+        self.selected_oois = sorted(list(set(request.GET.getlist("ooi", []))))
```

Comment:
Since sets are already iterables, there's no need to transform this into a list first
```suggestion
        self.selected_oois = sorted(set(request.GET.getlist("ooi", [])))
```